### PR TITLE
 data: Ketankham.json

### PR DESCRIPTION
**Pull request by Ketankham**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5076"></a>

### CODE REVIEW

File: Ketankham.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (WillBeebe) [inline Ketankham.json:1]: Delete this file.

File: data/Ketankham.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "bio": "A research scholar turned no code developer, machine learning enthusiast and looking to make open source contributions",
+  "avatar": "https://github.com/Ketankham.png",\
+    "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/KKhamgaonkar",
+      "icon": "FaTwitter"
+    }
+      ]
+}
```
Comment (WillBeebe) [inline data/Ketankham.json:14]:
```suggestion
{
   "name":"Ketan Khamgaonkar",
   "displayStatspublic":true,
   "type":"personal",
   "bio":"A research scholar turned no code developer, machine learning enthusiast and looking to make open source contributions",
   "avatar":"https://github.com/Ketankham.png",
   "links":[
      {
         "name":"Follow me on Twitter",
         "url":"https://twitter.com/KKhamgaonkar",
         "icon":"FaTwitter"
      }
   ]
}
```

**Review by WillBeebe — COMMENTED** [review]
Cheers

File: data/Ketankham.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Ketan Khamgaonkar",
+  "displayStatspublic":true,
```
Comment (SaraJaoude) [inline data/Ketankham.json:3]: Following a recent update this is no longer needed:
```suggestion
```

File: data/Ketankham.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "type": "personal",
+  "bio": "A research scholar turned no code developer, machine learning enthusiast and looking to make open source contributions",
+  "avatar": "https://github.com/Ketankham.png",\
```
Comment (SaraJaoude) [inline data/Ketankham.json:6]: Following a recent update this is no longer needed:
```suggestion
```

File: data/Ketankham.json (hunk 2 above)
Comment (DaduDev) [reply data/Ketankham.json:14]: You don't need displayStatsPublic and avatar, it was actually used in old models

File: data/Ketankham.json (hunk 4 above)
Comment (WillBeebe) [reply data/Ketankham.json:6]: Yanked

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]
Hello! It appears you pushed your file twice in 2 locations.  You will want to remove the one that is not in the data folder location

**Comment by Ketankham** [comment]
Sorry I did it by mistake .
I did lot of mistake even after pushing it twice.
I closed the pr accidently after that
Sorry for all the trouble I am creating here.



Regards,
Ketan

**Comment by Ketankham** [comment]
> Hello! It appears you pushed your file twice in 2 locations. You will want to remove the one that is not in the data folder location



> Hello! It appears you pushed your file twice in 2 locations. You will want to remove the one that is not in the data folder location

I have deleted one of the file
Am I done ?

**Comment by Ketankham** [comment]
> 

I have deleted one of the file
Am I done ?

**Comment by amandamartin-dev** [comment]
> Am I done ?

Hello! You are close and no worries you are not troubling any of us. We are all here to learn.

Two things to check:
You will see that @SaraJaoude made an edit for you to approve to remove a public stats option that is no longer needed. For that:
Click "commit suggestion" if you agree
Then "Resolve conversation"

You also have a warning that this branch is out of date with main which you will want to update

That should be everything! You are almost there

**Comment by Ketankham** [comment]
> 

Thank you so much for all the support and being patient throughout.

**Review by amandamartin-dev — APPROVED** [review]

**Pull request merged**